### ISSUE TITLE
"const override" is not correctly highlighted

### ISSUE THREAD

**Issue by adl** [ludwigpacifici/modern-cpp-font-lock#7]
Thank you for this package.

Using `c++-mode` and `modern-c++-font-lock-mode` the following code

``` c++
struct b : public a
{
  void foo() override;
  void foo() const override;
};
```

is font-locked as

![comp](https://cloud.githubusercontent.com/assets/822900/19392333/a7ff7fba-9230-11e6-8b6f-144fa0204a6e.png)

as if the second `override` was not detected as a keyword.

Would that be easy to fix?

**Comment by ludwigpacifici** [comment]
Hello @adl 

Thank you for your feedback.

I am not able to reproduce your issue. Here is what I get on my side:

![screenshot from 2016-10-15 09-50-48](https://cloud.githubusercontent.com/assets/923945/19408763/da32a804-92bc-11e6-829d-f7f3b367f423.png)

On the left, it is my personal emacs running `c++-mode` and `modern-c++-font-lock-mode`. On the right it is `emacs -Q` (it is a bare emacs with no configuration) with only `c++-mode` is loaded.

![screenshot from 2016-10-15 09-57-48](https://cloud.githubusercontent.com/assets/923945/19408796/90461e50-92bd-11e6-9804-007c820a975a.png)

Finally the last one is emacs running  `text-mode` (it puts away all syntax highlighting) and `modern-c++-font-lock-mode`.

From this, I understand that `c++-mode` and `modern-c++-font-lock-mode` both support `const` and `override`.

Can you tell me which version of emacs do you use? Also can you try to do like me (turning on and off major/minor modes) and show me what you see?

**Comment by adl** [comment]
Hmm, interesting.  For me loading `modern-c++-font-lock-mode` does make a difference.

`M-x version` says `GNU Emacs 24.5.1 (x86_64-pc-linux-gnu, GTK+ Version 3.21.5) of 2016-09-05 on trouble, modified by Debian`.

When I try `emacs -Q foo.cc` I get:

![emacs-q](https://cloud.githubusercontent.com/assets/822900/19408911/8b6d90b0-92c8-11e6-8abb-719a763c0a6d.png)

If I then manually load `modern-c++-font-lock-mode` this changes to:

![emacs-qm](https://cloud.githubusercontent.com/assets/822900/19408916/a084f772-92c8-11e6-9fc6-b557af09be8c.png)

Apparently emacs 25.1 landed two days ago in Debian, so I'll upgrade and see if that helps.

**Comment by adl** [comment]
Ok, so this is actually [fixed by Emacs 25.1](http://git.savannah.gnu.org/cgit/emacs.git/commit/lisp/progmodes/cc-langs.el?h=emacs-25.1&id=948fa912de164a1374c87e9206cddca741b7fa33).

Sorry for the noise!

**Comment by ludwigpacifici** [comment]
It's good you can upgrade your Emacs with the last version. You will have a newer version of the `c++-mode`. I expect `modern-c++-font-lock-mode` to support it as well.

Enjoy your coding!
